pygmt/datasets/load_remote_dataset.py

```diff
@@ -40,10 +40,8 @@ class GMTRemoteDataset(NamedTuple):
 
     Attributes
     ----------
-    title : str
-        The title of the dataset, used in error messages.
     name : str
-        The name assigned as an attribute to the DataArray.
+        The name assigned as an attribute to the DataArray and used in error messages.
```

Comment:
Prefer to remove it.
```suggestion
        The name assigned as an attribute to the DataArray.
```